app-text/pdfjam/pdfjam-3.12.ebuild

```diff
@@ -29,6 +29,8 @@ RDEPEND="
 "
 BDEPEND="test? ( app-arch/unzip )"
 
+DOC=( README.md )
+
```

Comment:
This is not needed. In fact, it is wrong on several levels:
- The package manager variable is named `DOCS`.
- There already is an explicit `dodoc README.md` in `src_install`.
- `DOCS` would only be used with the default `src_install` or when `einstalldocs` was called.
- Even if all of the above were fulfilled, `README*` is part of the `einstalldocs` defaults.

https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html#default-src_install

Reply:
I wouldn't bother with it. Only use case for explicit `DOCS` is when either there is documentation not covered by the defaults, or when the defaults would install unwanted files.